schema/rusk.proto

```diff
@@ -43,8 +36,16 @@ message GetProvisionersResponse {
     repeated Provisioner provisioners = 1;
 }
 
+message GetStateRootRequest {}
+
+message GetStateRootResponse {
```

Comment:
Did you manage to find a spec explaining more about State Root Hash so that we can include it in Block Header as well? What is its size?